state/execution.go

```diff
@@ -334,7 +355,7 @@ func updateValidators(currentSet *types.ValidatorSet, abciUpdates []abci.Validat
 
 // updateState returns a new State updated according to the header and responses.
 func updateState(state State, blockID types.BlockID, header *types.Header,
-	abciResponses *ABCIResponses) (State, error) {
+		abciResponses *ABCIResponses) (State, error) {
```

Comment:
If we reformat this than one param per line wtih trailing comma and newline after the last param.